### ISSUE TITLE
Broken Icons / Badges in Tooltips

### ISSUE THREAD

**Issue by midnight-dev** [kisstkondoros/csstriggers#14]
When hovering over a style rule in CSS, JS, or HTML files with a trigger flag tied to it (layout, composite, or paint) such as `color` or `height`, the images or icons inside of the tooltip are broken.

![image](https://user-images.githubusercontent.com/47105073/82716864-59c30280-9c5f-11ea-9a24-15cb810a3762.png)

This has been an issue for at least a few months, but maybe longer. Verified on multiple systems. Currently running VS Code 1.45.1. It seems to be broken on any theme, but the one I use on all of my machines is Monokai Dimmed. It definitely _used_ to work, so it was either a change in VS Code or a regression in the most recent updates of this extension. The identifier icons on the side of the code still show up, just not in the tooltips.

> Version: 1.45.1 (system setup)
> Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
> Date: 2020-05-14T08:27:35.169Z
> Electron: 7.2.4
> Chrome: 78.0.3904.130
> Node.js: 12.8.1
> V8: 7.8.279.23-electron.0
> OS: Windows_NT x64 10.0.18363

**Comment by midnight-dev** [comment]
Since e4ba3b2b293664c6e370571ba319bf1897fb78a6 was the last change to `triggerdecorator.ts` nearly a year ago, it's probably some change to VS Code that broke this.

`images/composite_paint_layout_wide.svg` and the other wide svg files still exist in the right location, images being held in the root extension path, so it's not like webpack threw them out.

Maybe it's this, just after L200 in triggerdecorator?

```js
// The markdown path parser under windows escapes the `userhome\.vscode` folder as `userhome.vscode`
if (process.platform === "win32") {
	path = path.replace("\\.", "\\\\.");
}
```

**Comment by kisstkondoros** [comment]
Thanks for the initial investigation, I'll look into this in more detail soon

**Comment by kisstkondoros** [comment]
This should be fixed now in the new version (v0.13.1)
